src/App.tsx

```diff
@@ -1,10 +1,47 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import React, { useEffect, useMemo, useState } from 'react';
 
-const USER_ID = 0;
+import { getTodos } from './api/todos';
+
+import { UserWarning } from './components/UserWarning/UserWarning';
+import { Header } from './components/Header/Header';
+import { TodoList } from './components/TodoList/TodoList';
+import { Footer } from './components/Footer/Footer';
+import { ErrorNotification } from './components/ErrorNotification';
+
+import type { Todo } from './types/Todo';
+import { StatusFilter } from './types/StatusFilter';
+import { ErrorMessage } from './types/ErrorMessage';
+
+const USER_ID = 12039;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [errorMessage, setErrorMessage] = useState<ErrorMessage | null>(null);
+  const [statusFilter, setStatusFilter] = useState(StatusFilter.ALL);
+
+  useEffect(() => {
+    getTodos(USER_ID)
+      .then(setTodos)
+      .catch(() => {
+        setErrorMessage(ErrorMessage.LOAD_ERROR);
+        setTimeout(() => setErrorMessage(null), 3000);
```

Comment:
Move this `setTimeout` to `ErrorNotification` component, as you have to do the same for all kinds of errors. It should react to any error and hide it in a 3-second